ui/app/AppLayouts/Wallet/views/TransactionDetailView.qml

```diff
@@ -38,96 +40,360 @@ Item {
+                        address: {
+                            if (!root.isTransactionValid)
+                                return ""
+                            switch(transactionHeader.transactionType) {
+                            case TransactionDelegate.Swap:
+                                return transaction.contract
+                            case TransactionDelegate.Bridge:
+                                return "" // TODO fill swap token's contract address for 'to' network for Bridge
+                            default:
+                                return ""
+                            }
+                        }
+                        symbol: {
+                            if (!root.isTransactionValid)
+                                return ""
+                            switch(transactionHeader.transactionType) {
+                            case TransactionDelegate.Swap:
+                                return d.swapSymbol
+                            case TransactionDelegate.Bridge:
+                                return transaction.value.symbol.toUpperCase()
+                            default:
+                                return ""
+                            }
+                        }
+                        networkName: d.bridgeNetworkFullname
+                        shortNetworkName: d.bridgeNetworkShortName
+                    }
+                }
 
-                type: StatusExpandableItem.Type.Tertiary
-                expandable: true
-                primaryText: qsTr("Transaction summary")
-                expandableComponent: transactionSummary
-                separatorVisible: false
-                expanded: true
+                Item {
+                    width: parent.width
+                    height: Style.current.bigPadding
+                }
+
+                DetailsPanel {
+                    width: progressBlock.width
+                    RowLayout {
+                        width: parent.width
+                        height: Math.max(implicitHeight, 85)
+                        spacing: 0
+                        TransactionDataTile {
+                            id: multichainNetworksTile
+                            Layout.fillHeight: true
+                            Layout.fillWidth: true
+                            title: qsTr("Networks")
+                            visible: d.multichainNetworks.length > 0
+                            Row {
+                                anchors {
+                                    top: parent.top
+                                    topMargin: multichainNetworksTile.statusListItemTitleArea.height + multichainNetworksTile.topPadding
+                                    left: parent.left
+                                    leftMargin: multichainNetworksTile.leftPadding
+                                }
+                                spacing: -4
```

Comment:
Wondering what this is compensating for.

Reply:
This 
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=12530-324879&t=vQm1swikL22aOYom-4

![image](https://github.com/status-im/status-desktop/assets/11396062/4e0bab5d-eef7-465b-a7d9-74aba8d26d4d)